### PR TITLE
Feature/jn1 1494 renewed

### PR DESCRIPTION
**Pull request by CRMTravis**

### CODE REVIEW

File: force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ **/
+public with sharing class NoContactMadeNotifierSchedulable implements Schedulable {
+  public Datetime currentTime = System.now();
+  /**
+   * Enqueue queueable class.
+   * @author Travis Allen | 06-22-2021
+   * @param ctx
+   **/
+  public void execute(SchedulableContext ctx) {
+    System.debug('Schedulable Class reached');
```
Comment (RemarioCRM) [inline force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls:19]: remove

File: force-app/main/default/classes/TestNoContactMadeNotifier.cls [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+  //Get Sales Team and SalesDeveloperTeam Queue Ids
+  public static Id tierOneQueueId = Util.getQueueId(Constants.TIER_ONE_QUEUE);
+  public static Id tierTwoQueueId = Util.getQueueId(Constants.TIER_TWO_QUEUE);
+
+  /**
+   * Data setup for testing
+   * @author Travis Allen | 06-25-2021
+   **/
+  @TestSetup
+  static void makeData() {
```
Comment (RemarioCRM) [inline force-app/main/default/classes/TestNoContactMadeNotifier.cls:32]: what is the current test coverage?

File: force-app/main/default/classes/NoContactMadeNotifierQueueable.cls [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    this.currentTime = currentTime;
+  }
+  /**
+   * Execute Queueable class
+   * @author Travis Allen | 06-22-2021
+   * @param context
+   **/
+  public void execute(QueueableContext context) {
+    List<Lead> leads = new List<Lead>(checkLeadsToBeNotified(this.currentTime));
+    update leads;
```
Comment (RemarioCRM) [inline force-app/main/default/classes/NoContactMadeNotifierQueueable.cls:29]: use sobject decision

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/TestNoContactMadeNotifier.cls (hunk 2 above)
Comment (RemarioCRM) [reply force-app/main/default/classes/TestNoContactMadeNotifier.cls:32]: ?

File: force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls (hunk 1 above)
Comment (RemarioCRM) [reply force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls:19]: ?

File: force-app/main/default/classes/NoContactMadeNotifierQueueable.cls (hunk 3 above)
Comment (RemarioCRM) [reply force-app/main/default/classes/NoContactMadeNotifierQueueable.cls:29]: ?

File: force-app/main/default/classes/TestNoContactMadeNotifier.cls (hunk 2 above)
Comment (CRMTravis) [reply force-app/main/default/classes/TestNoContactMadeNotifier.cls:32]: Current Unit test 89%

File: force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls (hunk 1 above)
Comment (RemarioCRM) [inline force-app/main/default/classes/NoContactMadeNotifierSchedulable.cls:19]: remove debug

File: force-app/main/default/classes/TestNoContactMadeNotifier.cls [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+   * Scenario 1: Lead has been reassigned to tier 1 for 8hrs without contact being made
+   * @author Travis Allen | 06-25-2021
+   **/
+  @isTest
+  public static void shouldNotifyNoContactMadeScenario1() {
+    //Get user from Tier 1 queue
+    User tierOneAgent = [
+      SELECT Id
+      FROM User
+      WHERE UserName = 'travistest@email.com'
```
Comment (RemarioCRM) [inline force-app/main/default/classes/TestNoContactMadeNotifier.cls:75]: please use more random names,  travistest@email.com will result in the same duplicate names in prod. something like travistest@dev2.com, try using the current org name.

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

**Review by RemarioCRM — APPROVED** [review]

**Pull request merged**